miden-tx/src/errors/tx_kernel_errors.rs

```diff
@@ -1,178 +1,151 @@
-pub const ERR_NON_FUNGIBLE_ASSET_FORMAT_POSITION_ONE_MUST_BE_FUNGIBLE_FAUCET_ID: u32 = 131127;
-pub const ERR_NON_FUNGIBLE_ASSET_FORMAT_HIGH_BIT_MUST_BE_ZERO: u32 = 131128;
-pub const ERR_FUNGIBLE_ASSET_MISMATCH: u32 = 131129;
-pub const ERR_NON_FUNGIBLE_ASSET_MISMATCH: u32 = 131130;
-pub const ERR_ACCOUNT_NONCE_INCR_MUST_BE_U32: u32 = 131131;
-pub const ERR_ACCOUNT_INSUFFICIENT_ONES: u32 = 131132;
-pub const ERR_ACCOUNT_SET_CODE_ACCOUNT_MUST_BE_UPDATABLE: u32 = 131133;
-pub const ERR_ACCOUNT_SEED_DIGEST_MISMATCH: u32 = 131134;
-pub const ERR_ACCOUNT_INVALID_POW: u32 = 131135;
-pub const ERR_NOTE_DATA_MISMATCH: u32 = 131136;
-pub const ERR_ASSET_NOT_FUNGIBLE_ID: u32 = 131137;
-pub const ERR_ASSET_INVALID_AMOUNT: u32 = 131138;
-pub const ERR_ASSET_NOT_NON_FUNGIBLE_ID: u32 = 131139;
-pub const ERR_INVALID_NOTE_TYPE: u32 = 131140;
-pub const ERR_NOTE_INVALID_TYPE_FOR_TAG: u32 = 131141;
-pub const ERR_NOTE_TAG_MUST_BE_U32: u32 = 131142;
-pub const ERR_SETTING_NON_VALUE_ITEM_ON_VALUE_SLOT: u32 = 131143;
-pub const ERR_SETTING_MAP_ITEM_ON_NON_MAP_SLOT: u32 = 131144;
-pub const ERR_READING_MAP_VALUE_FROM_NON_MAP_SLOT: u32 = 131145;
-pub const ERR_PROC_NOT_PART_OF_ACCOUNT_CODE: u32 = 131146;
-pub const ERR_PROC_INDEX_OUT_OF_BOUNDS: u32 = 131147;
-pub const ERR_ACCT_CODE_HASH_MISMATCH: u32 = 131148;
-pub const ERR_ACCT_TOO_MANY_PROCEDURES: u32 = 131149;
-pub const ERR_STORAGE_SLOT_INDEX_OUT_OF_BOUNDS: u32 = 131150;
-pub const ERR_INVALID_FAUCET_STORAGE_OFFSET: u32 = 131151;
-pub const ERR_NOTE_FUNGIBLE_MAX_AMOUNT_EXCEEDED: u32 = 131152;
-pub const ERR_NON_FUNGIBLE_ASSET_ALREADY_EXISTS: u32 = 131153;
-pub const ERR_INVALID_NOTE_IDX: u32 = 131154;
-pub const ERR_KERNEL_PROCEDURE_OFFSET_OUT_OF_BOUNDS: u32 = 131155;
-pub const ERR_CURRENT_ACCOUNT_IS_NOT_NATIVE: u32 = 131156;
-pub const ERR_INVALID_TX_EXPIRATION_DELTA: u32 = 131157;
+pub const ERR_ACCOUNT_CODE_COMMITMENT_MISMATCH: u32 = 0x0002004c;
+pub const ERR_ACCOUNT_CODE_IS_NOT_UPDATABLE: u32 = 0x0002003d;
+pub const ERR_ACCOUNT_INSUFFICIENT_NUMBER_OF_ONES: u32 = 0x0002003c;
+pub const ERR_ACCOUNT_NONCE_DID_NOT_INCREASE_AFTER_STATE_CHANGE: u32 = 0x00020009;
+pub const ERR_ACCOUNT_NONCE_INCREASE_MUST_BE_U32: u32 = 0x0002003b;
+pub const ERR_ACCOUNT_POW_IS_INSUFFICIENT: u32 = 0x0002003f;
+pub const ERR_ACCOUNT_READING_MAP_VALUE_FROM_NON_MAP_SLOT: u32 = 0x00020049;
+pub const ERR_ACCOUNT_SEED_DIGEST_MISMATCH: u32 = 0x0002003e;
+pub const ERR_ACCOUNT_SETTING_MAP_ITEM_ON_NON_MAP_SLOT: u32 = 0x00020048;
+pub const ERR_ACCOUNT_SETTING_VALUE_ITEM_ON_NON_VALUE_SLOT: u32 = 0x00020047;
+pub const ERR_ACCOUNT_STORAGE_COMMITMENT_MISMATCH: u32 = 0x00020050;
+pub const ERR_ACCOUNT_TOO_MANY_PROCEDURES: u32 = 0x0002004d;
+pub const ERR_ACCOUNT_TOO_MANY_STORAGE_SLOTS: u32 = 0x0002004f;
+pub const ERR_ACCOUNT_TOTAL_ISSUANCE_PROC_CAN_ONLY_BE_CALLED_ON_FUNGIBLE_FAUCET: u32 = 0x00020001;
+pub const ERR_CURRENT_ACCOUNT_IS_NOT_NATIVE: u32 = 0x00020054;
```

Comment:
Let's rename this error to `ERR_ACCOUNT_IS_NOT_NATIVE` to match the pattern where the origin is the first word in the error name.